ast.go

```diff
@@ -0,0 +1,93 @@
+package sshconfigo
+
+// String is a string inside a line. It does not contain new line endings.
+type String struct {
+	// Pos is the position of the first character in the line.
+	Pos int
+	// Text is the contents of the string.
+	Text string
+}
+
+// Line is a line inside a ssh_config file. An empty line is represented by a
+// Line that does not contain a value for the keyword.
```

Comment:
With `keyword` you mean `LiteralKeyword` right?

Reply:
Yes, of course. LiteralKeyword is the field that contains the keyword.

Reply:
Perfect!